src/inputs.ts

```diff
@@ -52,16 +55,23 @@ export class ActionConfig {
   public readonly svgoOptionsPath: string;
 
   constructor(config: RawActionConfig = { }) {
+    this.comment = ActionConfig.getCommentValue(config);
     this.commitDescription = ActionConfig.getCommitDescription(config);
     this.commitTitle = ActionConfig.getCommitTitle(config);
-    this.enableComments = ActionConfig.getCommentsValue(config);
+    this.enableComments = ActionConfig.getEnableCommentsValue(config);
     this.ignoreGlob = ActionConfig.getIgnoreGlob(config);
     this.isDryRun = ActionConfig.getDryRunValue(config);
     this.svgoOptionsPath = ActionConfig.getSvgoOptionsPath(config);
   }
 
-  private static getCommentsValue(config: RawActionConfig): boolean {
-    return this.normalizeBoolOption(config.comments, INPUT_NAME_COMMENTS, true);
+  private static getCommentValue(config: RawActionConfig): string {
+    const value = (config.comments !== undefined) ?
+      config.comments : core.getInput(INPUT_NAME_COMMENTS, NOT_REQUIRED);
+    if (typeof value === STRING && value !== TRUE) {
+      return value as string;
```

Comment:
Add a comment clarifying why we allow the value to be (the string) `"false"`

```suggestion
      // If value is (the string) `"false"`, comments will be disabled so it does not matter that
      // the comment template is `"false"`. If the value is (the string) `"true"`, we interpret it
      // as (the boolean) `true`, so the default template should be used.
      return value as string;
```